src/transformation/vhacd/vhacd.rs

```diff
@@ -239,12 +240,18 @@ impl VHACD {
                     &mut right_ch_pts,
                     &mut left_ch_pts,
                 );
-                right_ch = convex_hull(&right_ch_pts);
-                left_ch = convex_hull(&left_ch_pts);
+                right_ch = convex_hull(&right_ch_pts)
+                    .unwrap_or_else(ConvexHullError::into_incorrect_empty);
```

Comment:
I'm not a fan of this behavior, but I think it makes sense to have an indicator at call site that the return may be incorrect (empty).